blueprints/30-nodes/blueprint.py

```diff
@@ -331,7 +331,8 @@ def build_node(manifest, node_coord):
     :param 'node_coord': [int\str] node number or name.
     :return: flask's response data.
     """
-    golden_tar = BP.config['GOLDEN_IMAGE']
+    #golden_tar = BP.config['GOLDEN_IMAGE']
+    golden_tar = BP.config.golden_image
```

Comment:
I know that this reflects the previous bad practice, but should the attribute also be named "dot golden_tar" ?  I won't object if you keep it the way it is, but since you're making this change....

Reply:
actually, this line:

" golden_tar = BP.config.golden_image

should be

golden_tar = BP.config["GOLDEN_IMAGE"]

Therefore question: do you want "GOLDEN_IMAGE" to become "GOLDEN_TAR"?

Reply:
Yes I do.